packages/grid/x-data-grid/src/hooks/features/editRows/gridEditRowsSelector.ts

```diff
@@ -1,9 +1,4 @@
 import { GridStateCommunity } from '../../../models/gridStateCommunity';
 
-// TODO v6: rename to gridEditingStateSelector
+// TODO v6: rename to gridEditRowsStateSelector
```

Comment:
Remove the `TODO` ? The current name equals the new one

Reply:
My Replace All failed.